### ISSUE TITLE
LightGallery Preview Issue - IE Only

### ISSUE THREAD

**Issue by dbradley94** [sachinchoolur/lightGallery#754]
When you click on an image, the preview image appears, disappears then reappears again. I've only found this on IE. Works fine on all other browsers.
Version: IE 11

**Comment by sachinchoolur** [comment]
Hey @dbradley94,
It workes fine for me. Which version of OS do you use?

**Comment by dbradley94** [comment]
Hi @sachinchoolur
I'm using Windows 10. A few of my colleagues are experiencing the same issue as well

**Comment by basticom-internet** [comment]
This is still an issue. The #784 fix by peppertom seems to be a partly fix, where all transitions are removed. Zooming and navigating images using the arrows is broken (read: not animated) when this fix is applied. 

I have tried some `backface-visibility` and `translateZ` hacks, but none worked. Strangely enough, this bug only appears in IE11 on Windows 10. When testing in IE11 on Windows 8, there are no issues. Even on IE11 on Windows 7 the transitions are working fine. By the looks of it, the rendering on Windows 10 differs from older Windows versions and that is what is causing the problem. 

I have captured the behavior of both versions (IE11 @ Windows 7 and IE11 @ Windows 10) in this video:
https://youtu.be/BcHXllELZIg

Not sure what to do next...
